src/cartesian/CartesianAxis.tsx

```diff
@@ -241,9 +241,21 @@ export class CartesianAxis extends Component<Props, IState> {
     let tickItem;
 
     if (React.isValidElement(option)) {
-      tickItem = React.cloneElement(option, props);
+      const customElementProps = option.props || {};
+      const existingCustomClassName = get(customElementProps, 'className');
```

Comment:
why is this `option.props` and the function equivalent is `props`? It should be one or the other in both cases right?